### PR TITLE
Joint liquidity mining proposal and KIP-10

### PR DESCRIPTION
**Pull request by p-thudo**

### CODE REVIEW

**Comment by tranvictor** [comment]
This PR is good to go so please change it to be KIP-10 instead of 11  @p-thudo

**Review by tranvictor — APPROVED** [review]

**Review by Anyhowclick — APPROVED** [review]

File: KIPs/kip-11.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+Once the proposal has been submitted on this [Github form](https://github.com/p-thudo/KIPs/issues/new?assignees=&labels=proposal%2Ctriage&template=joint_mining_proposal.yml&title=%5BProposal+from%5D%3A+) and the [forum](https://gov.kyber.org/c/collaborations), the Kyber core team will discuss it with the community. If the Kyber core team determines that the project has met the requirements stipulated in the liquidity mining framework, an on-chain proposal will be made for voting by the KyberDAO on kyber.org. The entire process takes approximately **14 days** from the start of the proposal submission.
```
Comment (loiluu) [inline KIPs/kip-11.md]: replace "Kyber core team" by "Kyber community"

File: KIPs/kip-11.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+| Project reputation, quality, and security                                                                                                                          | Since Rainmaker will encourage more users to purchase the project tokens and adopt related products, it is ideal to collaborate with the best and most promising projects in DeFi.                                                                                                                                                                                                                                                                                                                                                                                    |
```
Comment (loiluu) [inline KIPs/kip-11.md]: Rainmaker doesn't encourage the purchase of new project tokens. It encourages more liquidity contribution to the project tokens.

File: KIPs/kip-11.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+| Ability to at least match the KNC incentives provided by Kyber Network. Minimum of $200,000 worth of incentives in total from both parties. | Both Kyber and the project should commit a meaningful amount of incentives in order to ensure an attractive APY and amplify the impact of the Rainmaker program. The project should indicate a ballpark range of token incentives they are willing to contribute. KNC reward pool will depend on whether it is a new or established token: Kyber can cover from 30%, 50%, or up to 1:1 of the project’s contribution, depending on how established the token is. On top of KNC rewards, Kyber will provide additional market budget and support to promote the campaign. |
```
Comment (loiluu) [inline KIPs/kip-11.md]: whats "market budget"?

File: KIPs/kip-11.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+### Example
+
+**1. Project Introduction**
+XYZ Finance is a DeFi project that...
+
+**2. Proposal Summary, Motivation and Key Details**
+*Refer to the selection criteria guide below.*
+
+XYZ Finance is proposing a joint liquidity mining program with Kyber to help improve liquidity for the XYZ token on the capital efficient KyberDMM protocol.
```
Comment (loiluu) [inline KIPs/kip-11.md]: remove "capital efficient"

File: KIPs/kip-11.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+During the campaign period, liquidity providers who add liquidity (deposit tokens) on the eligible XYZ-ETH pool will receive LP tokens that can be staked on the Yield page for additional XYZ and KNC token rewards. As part of the campaign launch, there will also be joint marketing efforts with Kyber Network.
```
Comment (loiluu) [inline KIPs/kip-11.md]: is it the pool fixed? like just XYZ-ETH pool?

**Review by loiluu — CHANGES_REQUESTED** [review]
please make the changes according to my comments. some points also need further clarification.

File: KIPs/kip-11.md (hunk 3 above)
Comment (p-thudo) [reply KIPs/kip-11.md]: Funding from DAO community or from new mining KNC proposal (if it is approved by DAO community). Any suggestions for changing content?

File: KIPs/kip-11.md (hunk 5 above)
Comment (p-thudo) [reply KIPs/kip-11.md]: Partners should have options on which token they want to pair with in the rewarded pool. This also should be asked in the Joint LM proposal form. Please consult on list of token option @tranvictor.

Comment (tranvictor) [reply KIPs/kip-11.md]: ETH, WBTC, USDT, USDC, DAI and KNC are the options we can propose.

**Review by loiluu — APPROVED** [review]

**Pull request merged**